postcss.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  syntax: 'postcss-scss',
+  parser: 'postcss-safe-parser',
+  plugins: [
+    require('autoprefixer'),
+    require('postcss-import'),
+    require('postcss-uncss')({
```

Comment:
I put this to purge the styles not used, but I think this significantly slows down the dev server. I will need to put some conditions to use it only for production build